### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by chenxshani**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: datasets/humor-patterns.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Name: Humor patterns
+Description: Patterns used for quering Alexa traffic when creating the taxonomy described in the paper "“Alexa, Do You Want to Build a Snowman?” Characterizing Playful Requests to Conversational Agents"  by C. Shani, A. Libov, S. Tolmach, L. Lewin-Eytan, Y. Maarek, and D. Shahaf (CHI LBW 2022)
+Contact: alibov@amazon.com ########## Alex?
+Tags:
```
Comment (jflasher) [inline datasets/humor-patterns.yaml:4]: @chenxshani please use approved tags from the list at https://github.com/awslabs/open-data-registry/blob/main/tags.yaml

File: datasets/humor-patterns.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Name: Humor patterns
+Description: Patterns used for quering Alexa traffic when creating the taxonomy described in the paper "“Alexa, Do You Want to Build a Snowman?” Characterizing Playful Requests to Conversational Agents"  by C. Shani, A. Libov, S. Tolmach, L. Lewin-Eytan, Y. Maarek, and D. Shahaf (CHI LBW 2022)
```
Comment (jflasher) [inline datasets/humor-patterns.yaml:2]: Please add a bit more of a description about the data here, what is it?

File: datasets/humor-patterns.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Name: Humor patterns
+Description: Patterns used for quering Alexa traffic when creating the taxonomy described in the paper "“Alexa, Do You Want to Build a Snowman?” Characterizing Playful Requests to Conversational Agents"  by C. Shani, A. Libov, S. Tolmach, L. Lewin-Eytan, Y. Maarek, and D. Shahaf (CHI LBW 2022)
+Contact: alibov@amazon.com ########## Alex?
+Tags:
+  - humor
+  - alexa
+License: There are no restrictions on the use of this data. ########## Verify
```
Comment (jflasher) [inline datasets/humor-patterns.yaml:7]: You will need to add an actual license here.

File: datasets/humor-patterns.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+Name: Humor patterns
+Description: Patterns used for quering Alexa traffic when creating the taxonomy described in the paper "“Alexa, Do You Want to Build a Snowman?” Characterizing Playful Requests to Conversational Agents"  by C. Shani, A. Libov, S. Tolmach, L. Lewin-Eytan, Y. Maarek, and D. Shahaf (CHI LBW 2022)
+Contact: alibov@amazon.com ########## Alex?
+Tags:
+  - humor
+  - alexa
+License: There are no restrictions on the use of this data. ########## Verify
+Resources:
+  - Description: Humor patterns
+    ARN: arn:aws:s3:::humor-patterns
```
Comment (jflasher) [inline datasets/humor-patterns.yaml:10]: Correct bucket name is amazon-humor-patterns

File: datasets/humor-patterns.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+Description: Patterns used for quering Alexa traffic when creating the taxonomy described in the paper "“Alexa, Do You Want to Build a Snowman?” Characterizing Playful Requests to Conversational Agents"  by C. Shani, A. Libov, S. Tolmach, L. Lewin-Eytan, Y. Maarek, and D. Shahaf (CHI LBW 2022)
+Contact: alibov@amazon.com ########## Alex?
+Tags:
+  - humor
+  - alexa
+License: There are no restrictions on the use of this data. ########## Verify
+Resources:
+  - Description: Humor patterns
+    ARN: arn:aws:s3:::humor-patterns
+    Region: us-east-1
```
Comment (jflasher) [inline datasets/humor-patterns.yaml:11]: Region should be us-west-2 I think.

File: datasets/humor-patterns.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+Name: Humor patterns
```
Comment (jflasher) [inline datasets/humor-patterns.yaml:1]: Please add a little more description to the name that'd help someone know what this data is.

**Review by jflasher — CHANGES_REQUESTED** [review]
Hi @chenxshani added a few inline comments. Also, please add a ManagedBy field like in https://github.com/awslabs/open-data-registry/blob/main/datasets/airborne-object-tracking.yaml#L5. 

Those changes should get us close to being able to merge. Thanks!

**Comment by chenxshani** [comment]
Addressed all comments. Please review. Thanks!

**Review by jflasher — CHANGES_REQUESTED** [review]
Hi @chenxshani you'll need to add `UpdateFrequency` and `Documentation` fields as well. You can see all the required fields in the table at https://github.com/awslabs/open-data-registry#how-are-datasets-added-to-the-registry.

**Comment by chenxshani** [comment]
Ready? :) @jflasher

**Pull request merged**